.gitignore

```diff
@@ -1,3 +1,6 @@
 *.exe
 vendor
 .DS_Store
+
+# Local IDE generated files.
+.vscode
```

Comment:
desktop.ini may be in this ignore file 🥹🥹🥹

Reply:
desktop.ini is so bad 🥺